projects/acs-engine/azure-purge/azure-secret.yaml

```diff
@@ -0,0 +1,12 @@
+### Secret was generated with command:
+### oc create secret generic azure-secret --from-literal=azure_client_id=xxxx --from-literal=azure_client_secret=xxxxx --dry-run -o yaml
```

Comment:
This needs to be part of the Makefile I think

@stevekuznetsov @smarterclayton @dobbymoodge how do we create secrets for other clouds in the CI cluster today? Related: https://trello.com/c/3LlnWrxf/550-ci-automate-secret-creation-in-the-ci-cluster